policy/templates/subtemplates/auto/auto-test.gotmpl

```diff
@@ -149,9 +151,9 @@
           python-version: '3.10'
       - name: Run API tests
         id: test_execution
-        working-directory: tests/{{ .test }}
+        working-directory: tyk-analytics/tests/{{ .test }}
         run: |
-          pytest="pytest --random-order --maxfail=3 --junitxml=${XUNIT_REPORT_PATH} --cache-clear --log-cli-level=ERROR"
+          pytest="pytest --ci --random-order --force-flaky --no-success-flaky-report --maxfail=3 --junitxml=${XUNIT_REPORT_PATH} --cache-clear --ignore=./tests/mdcb -v --log-cli-level=ERROR"
```

Comment:
Should we be ignoring mdcb tests?

Reply:
this does not ignore mdcb tests, mdcb tests are located into mdcb_tests folder. /test/mdcb is used to import some base clases and old tests